src/core/server/plugins/discovery/plugins_discovery.ts

```diff
@@ -138,7 +138,7 @@ function findManifestInFolder(
   return fsStat$(resolve(dir, 'kibana.json')).pipe(
     mergeMap((stats) => {
       // `kibana.json` exists in given directory, we got a plugin
-      if (stats.isFile()) {
+      if ((stats as Stats).isFile()) {
```

Comment:
I'm surprised we need this, but ok, we can refactor the code later

Reply:
Hmm that's odd. At some point while working on this PR, that line failed with a TypeScript error because it thought `stats` was of type `any`. But I see now this doesn't appear to be the case anymore. I don't think anything changed, but maybe my local environment was in a bad state at the time. I'll try and remove the two `Stats` type-casts in this file and see if CI passes.

Reply:
Similar to https://github.com/elastic/kibana/pull/61587#discussion_r404561127 btw

Reply:
Tests still pass after removing the casts. Nicely spotted 👍